### DISCUSSION TITLE
关于sm3、sm4密钥

### DISCUSSION THREAD

**Discussion by flagshen** [he3-app/he3-feedback#126] [Ideas]
现在sm3和sm4的密钥我看直接是输入的密钥字符串，对一些乱码密钥不适用，希望密钥加一个是否base64或者hex编码

**Comment by lyzhang1999** [comment]
感谢您的建议，我已经将意见反馈给产品部门，并建立了 issues 来进一步跟进这个问题：https://github.com/he3-app/he3-feedback/issues/127。

产品进度将在 issues 提供反馈。
